examples/vue-code-file/eslint.config.js

```diff
@@ -1,18 +1,31 @@
-import vueParser from 'vue-eslint-parser';
+import { mergeProcessors } from 'eslint-merge-processors';
+import pluginVue from 'eslint-plugin-vue';
+import processorVueBlocks from 'eslint-processor-vue-blocks';
 import js from '@eslint/js';
 import graphqlPlugin from '@graphql-eslint/eslint-plugin';
 
 export default [
   {
-    files: ['**/*.js', '**/*.vue'],
+    files: ['**/*.js'],
     processor: graphqlPlugin.processor,
     rules: js.configs.recommended.rules,
   },
+  ...pluginVue.configs['flat/recommended'],
   {
     files: ['**/*.vue'],
-    languageOptions: {
-      parser: vueParser,
-    },
+    // `eslint-plugin-vue` will set a default processor for `.vue` files
+    // we use `eslint-merge-processors` to extend it
+    processor: mergeProcessors([
+      pluginVue.processors['.vue'],
+      processorVueBlocks({
+        blocks: {
+          styles: false,
+          customBlocks: true,
+          script: true,
+          template: false,
```

Comment:
I think we can't declare graphql documents in template? can be removed? @bmulholland
```suggestion
```

Reply:
I suspect you _can_. But people _shouldn't_. It certainly wouldn't be the Vue way, or a best practice. I say don't support it.

Reply:
ok let's remove this from example